micah-akpan/questioner-react
# #164695740 Setup React project

#### What does this PR do?
Setups React Project
#### Description of Task to be completed?
Setup React project structure
#### How should this be manually tested?
- Clone repo
- Install dependencies `yarn install` or `npm install`
- Run application `yarn dev` or `npm run dev`
- You should see the homepage
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#164695740
#### Screenshots (if appropriate)
#### Questions:
